cmd/fyneterm/main.go

```diff
@@ -104,9 +111,10 @@ func newTerminalWindow(a fyne.App, th fyne.Theme, debug bool) fyne.Window {
 	t.SetDebug(debug)
 	setupListener(t, w)
 	w.SetContent(container.NewStack(bg, img, over, t))
-
 	cellSize := guessCellSize()
-	w.Resize(fyne.NewSize(cellSize.Width*80, cellSize.Height*24))
+	minSize := fyne.NewSize(cellSize.Width*DefaultCols, cellSize.Height*DefaultRows)
+	t.SetMinSize(minSize)
```

Comment:
This is not the same behaviour as before, it is setting the preferred size to be the minimum size - so now we cannot make the terminal any smaller.
Sounds bad for usage on mobile or even desktops with small space etc.

Reply:
I think the real issue is that we clear the cell data on a resize and we shouldn't we should just not render it

Reply:
Hmm, well the reason they are cleared is because we only want as many cells as shown on screen - so we don't leave the "ones off the end of the line" in memory for later - we re-use them on cells further down and clear out extras.
The algorithm could be improved to dis-use the ones from the end of the row instead of re-using and just removing those not in use at the end of the view?